--- kernel-cleaner/kernel-cleaner	2012-09-20 05:43:56.000000000 +0000
+++ some_trash/kernel-cleaner-no-parallel	2012-09-20 05:45:40.000000000 +0000
@@ -15,18 +15,16 @@
 ############################################################################
 ################ init ######################################################
 init(){
-cores="$(awk '/cores/ {print $4}' /proc/cpuinfo | uniq)"
-threads="$(( ${cores} + 1 ))"
 
 SRC="/usr/src/linux"
 
 echo "Kernels....";
-KERNELS="$(parallel -j${threads} "file -F ' '" ::: /boot/* | awk '/Linux.*version/ {print $1,$9}')" ; eend 0 2>/dev/null
+KERNELS="$(file -F ' ' /boot/* | awk '/Linux.*version/ {print $1,$9}')" ; eend 0 2>/dev/null
 echo "Initrams...";
-INITRAMS="$(parallel -j${threads} "file -F ' '" ::: /boot/* | awk '/compressed data/ {print $1,$2}')" ; eend 0 2>/dev/null
+INITRAMS="$(file -F ' ' /boot/* | awk '/compressed data/ {print $1,$2}')" ; eend 0 2>/dev/null
 echo "Configs....";
-parallel -j${threads} "file -F ' '" ::: /boot/* | grep -q 'ASCII' || \
-parallel -j${threads} "file -F ' '" ::: /boot/* | grep -q 'config build file' && \
+file -F ' ' /boot/* | grep -q 'ASCII' || \
+file -F ' ' /boot/* | grep -q 'config build file' && \
 CONFIGS="$(grep -H 'Linux/' /boot/* 2>/dev/null | awk '{print $1,$3}')"
 CONFIGS="${CONFIGS//:#/}" ; eend 0 2>/dev/null
 echo "Maps.......";
